cdap-ui/app/cdap/components/EntityCard/EntityCardHeader/index.js

```diff
@@ -44,16 +44,38 @@ export default class EntityCardHeader extends Component {
 
   render() {
     return (
-      <div
-        onClick={this.props.onClick}
-        className={classnames("entity-card-header", this.props.className)}
-      >
-        <h4>
-          <span className={classNames('entity-icon', this.props.entity.icon)}></span>
-          <span className="entity-type">
-            {T.translate(`commons.entity.${this.getType()}.singular`)}
-          </span>
-        </h4>
+      <div className="card-header-wrapper">
+        {
+          this.props.preferencesSaved ?
```

Comment:
We use EntityCard for all the entities in CDAP. A prop named `preferencesSaved` might not be the right name for it. May be `successMessage`?